src/blockstorage/sequential_files.cpp

```diff
@@ -216,7 +216,7 @@ void FindFilesToPruneSequential(std::set<int> &setFilesToPrune, uint64_t nLastBl
                 break;
             }
 
-            // don't prune files that could have a block within MIN_BLOCKS_TO_KEEP of the main chain's tip but keep
+            // don't prune files that could have a block within MIN_BLOCKS_TO_KEEP of the main chain's tipFrac but keep
             // scanning
```

Comment:
what's a tipFrac?

Reply:
tipFrac is the android studio doing a global search and replace across the ENTIRE damn code base rather than just some android code... I reverted almost everything but clearly missed this one.  Will fix.